etc/vim/ftplugin/chpl.vim

```diff
@@ -0,0 +1,16 @@
+" Disable repeating * in comments, because chpldoc doesn't understand them,
+" so that isn't the Chapel style.
+setlocal comments=s1:/*,ex:*/,://
+
+" Use list formatting at all
+setlocal formatoptions+=n
+" Allow 1. * - lists (for use in chpl-doc comments)
+setlocal formatlistpat=^\\s*\\(\\d\\+[\\]:.)}\\t\ ]\\\|[*-][\\t\ ]\\)\\s*
+
+" If you prefer spaces to tabs, uncomment the following:
+"  (To get a real tab with these settings, do CTRL-V TAB)
+setlocal expandtab       " replace indents with spaces
```

Comment:
I'm OK either way on the tab settings. Do you think it's better to leave those in .vimrc? I bet that these settings can be overridden (I think that's what setlocal does but I'm not sure)